identity/app/model/EmailSubscriptions.scala

```diff
@@ -307,7 +308,8 @@ object EmailSubscriptions {
       description = "For all you need to know about technology in the world this week, news, analysis and comment.",
       frequency = "Every Thursday",
       listId = "1902",
-      subscribedTo = subscribedListIds.exists{ x => x == "1902" }
+      subscribedTo = subscribedListIds.exists{ x => x == "1902" },
```

Comment:
BTW `subscribedListIds.contains("1902")` is equivalent and a bit easier on the eye.